gui.py

```diff
@@ -70,236 +89,273 @@ def wrap_in_section(content, section_classes="", container_classes="", div_class
+<svg viewBox="0 0 1000 1000" class="icon" height="1em" width="1em"><path d="m500 450c-83 0-150-67-150-150 0-83 67-150 150-150 83 0 150 67 150 150 0 83-67 150-150 150z m400 150h-120c-16 0-34 13-39 29l-31 93c-6 15-23 28-40 28h-340c-16 0-34-13-39-28l-31-94c-6-15-23-28-40-28h-120c-55 0-100-45-100-100v-450c0-55 45-100 100-100h800c55 0 100 45 100 100v450c0 55-45 100-100 100z m-400-550c-138 0-250 112-250 250 0 138 112 250 250 250 138 0 250-112 250-250 0-138-112-250-250-250z m365 380c-19 0-35 16-35 35 0 19 16 35 35 35 19 0 35-16 35-35 0-19-16-35-35-35z" transform="matrix(1 0 0 -1 0 850)"></path></svg>
+</span> icon in the upper-right of each chart to download it.</p>
+            """
+        )
     ],
+    section_classes="words-block-grey",
+    div_classes="content",
 )
 
-dropdown = dcc.Dropdown(
-    options=[
-        {"label": "New York City", "value": "NYC"},
-        {"label": "Montréal", "value": "MTL"},
-        {"label": "San Francisco", "value": "SF"},
-    ],
-    value="MTL",
-)
-dropdown_field = html.Div(
-    className="field",
-    children=[
-        html.Label("Dropdown", className="label"),
-        html.Div(className="control", children=[dropdown]),
-    ],
-)
 
-multi_dropdown = dcc.Dropdown(
-    options=[
-        {"label": "New York City", "value": "NYC"},
-        {"label": "Montréal", "value": "MTL"},
-        {"label": "San Francisco", "value": "SF"},
-    ],
-    multi=True,
-    value="MTL",
-)
+def remove_asos_awos(location):
+    """remove the ASOS and AWOS strings from station_name field,
+    for airports dropdown"""
+    location = location.replace("(ASOS)", "")
+    location = location.replace("(AWOS)", "")
+    return location
 
-multi_dropdown_field = html.Div(
-    className="field",
-    children=[
-        html.Label("Multi-dropdown", className="label"),
-        html.Div(className="control", children=[multi_dropdown]),
-    ],
+def format_location_name(station_name, sid):
+    """Uses new location names"""
+    try:
+        return luts.new_location_names[sid]
+    except KeyError:
+        return station_name
+
+airports_dropdown_field = wrap_in_field(
+    "Select an airport",
+    dcc.Dropdown(
+        id="airports-dropdown",
+        options=[
+            {"label": f"{format_location_name(remove_asos_awos(airport.station_name).title(), index)} / {airport.real_name} ({index})", "value": index}
+            for index, airport in luts.map_data.iterrows()
+        ],
+        value="PAFA",
+    ),
 )
 
-slider = dcc.Slider(min=-5, max=10, step=0.5, value=-3)
-slider_field = html.Div(
-    className="field",
-    children=[
-        html.Label("Slider", className="label"),
-        html.Div(className="control", children=[slider]),
-    ],
+
+map_selector_section = wrap_in_section(
+    html.Div(
+        children=[
+            airports_dropdown_field,
+            dcc.Graph(
+                id="map",
+                figure=map_figure,
+                config={"displayModeBar": False, "scrollZoom": True},
```

Comment:
_Usually_ I prefer to disable `scrollZoom` because of this: you're scrolling down the page, you happen to pause a scroll while your cursor is over the map, then you go to scroll again.  Expected: you scroll the page down.  Actual: you're zooming the map!!

But this isn't hard-and-fast.  If someone asked for this specifically, 🤷 if you happen to strongly like it here, 🤷 OK.